examples/node/Examples/quickstart.js

```diff
@@ -162,3 +163,70 @@ describe("QuickStart Local", () => {
     // :snippet-end:
   });
 });
+
+describe("Quickstart Sync", () => {
+
+  test.skip("should open a FS realm with initial subscriptions", async () => {
+    // :snippet-start: open-realm-with-subscriptions
+    // :snippet-start: anonymous-login
+    // :snippet-start: initialize
+    // :replace-start: {
+    //   "terms": {
+    //     "flexsyncjstest-smixl": "<yourAppId>"
+    //   }
+    // }
+    // Initialize your App.
+    const app = new Realm.App({
+      id: "flexsyncjstest-smixl",
+    });
+    // :replace-end:
+    // :snippet-end:
+    
+    // Authenticate an anonymous user.
+    await app.logIn(Realm.Credentials.anonymous());
+    // :snippet-end:
+
+
+    // Define an object model
+    const TaskSchema = {
+      name: "Task",
+      properties: {
+        _id: "int",
+        name: "string",
+        status: "string?",
+        progressMinutes: "int?",
+        owner: "string?",
+        dueDate: "date?",
+      },
+      primaryKey: "_id",
+    };
+    
+    // Create a `SyncConfiguration` object.
+    const config = {
```

Comment:
```suggestion
    const config = {
      schema: [TaskSchema],
```

must include the schema in the config.

Reply:
Included this locally. Will push in my next commit.